pkg/v1/tkg/test/tkgctl/shared/e2e_common.go

```diff
@@ -216,7 +216,7 @@ func E2ECommonSpec(ctx context.Context, inputGetter func() E2ECommonSpecInput) {
 				By(fmt.Sprintf("Verify workload cluster %q resources have been deleted", clusterName))
 				Eventually(func() bool {
 					return clusterResourcesDeleted(ctx, mngClient, clusterResources)
-				}, waitTimeout, pollingInterval).Should(BeTrue())
+				}, 2*waitTimeout, pollingInterval).Should(BeTrue())
```

Comment:
Why increasing the timeout, and what guarantees this is enough? Why current timeout used to work before?

Reply:
Hi @maralavi, the old timeout didn't work for Calico tests before. The test was flaky due to timeout in resource deletion. After increasing the timeout I didn't see that issue for several runs. So I think it helps mitigate the flaky scenario.